internal/webhook/external/gitea.go

```diff
@@ -0,0 +1,145 @@
+package external
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	gh "github.com/google/go-github/v71/github"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kargoapi "github.com/akuity/kargo/api/v1alpha1"
+	"github.com/akuity/kargo/internal/git"
+	xhttp "github.com/akuity/kargo/internal/http"
+	"github.com/akuity/kargo/internal/logging"
+)
+
+const (
+	GiteaSecretDataKey = "secret"
```

Comment:
No need to export this. Most of the receivers don't. The GitHub receiver does, but it's an exception. A few other packages use it. If you modeled this closely on that receiver, this is probably just copy/pasta.

```suggestion
	giteaSecretDataKey = "secret"
```